src/charts/no_margin_chart_mixin.coffee

```diff
@@ -0,0 +1,19 @@
+# Remove all extra margins so that graph elements can line up with other
+# elements more easily
+Ember.Charts.NoMarginChartMixin = Ember.Mixin.create
+  marginLeft: 0
+  marginRight: 0
+
+  #There should be no padding if there is no legend
+  marginBottom: Ember.computed ->
+    return 0 unless @get 'hasLegend'
+    return 30
+  .property 'hasLegend'
+
+  maxLabelLength: (SVGTextArray) ->
+    maxLabel = 0
+    SVGTextArray.each ->
```

Comment:
Weird as it seems (since SVG is all caps normally) I'd call this `svgTextArray` to make it clear it's an object/instance and not a class.